src/clr-angular/utils/drag-and-drop/draggable-ghost.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {animate, style, transition, trigger} from "@angular/animations";
+import {Component, ElementRef, HostBinding, NgZone, OnDestroy, Optional, Renderer2} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
+import {ClrDragEvent} from "./interfaces/drag-event";
+import {ClrDragEventListener} from "./providers/drag-event-listener";
+import {ClrDraggableStateRegistrar} from "./providers/draggable-state-registrar";
+
+const ANIMATION_DURATION = "0.2s ease-in-out";
```

Comment:
`ANIMATION_DURATION` is misleading. Has both the duration and the timing function. May be just call it `ANIMATION_PROPS` or a better name if you can think of one? I am really bad at this.

Reply:
oooh how about `ANIMATION_CONFIG`?

Reply:
I shouldn't have set this animation config to a variable like this. So I've inlined this in the animations property of the component.